### ISSUE TITLE
Game telling to download the newer version despite already having it

### ISSUE THREAD

**Issue by simonherrmann** [unknown-horizons/unknown-horizons#2730]
I've just downloaded and installed Unknown-Horizons-2017.2.158-Setup-VC14-x86.exe on Windows 10. After starting the game, it tells me that I only have 2017.1 and should get 2017.2 and it redirects me to the download site, which only offers the version I've just installed. What's wrong here?

**Comment by LinuxDonald** [comment]
Yeah that's an knowed bug. It will be fixed in the next version oh uh. Ignore the message for now.

**Comment by simonherrmann** [comment]
Great, thanks

**Comment by jakoch** [comment]
- Windows Installer doesn't set the version
- get_git_version() fails on all methods
  - ref. https://github.com/unknown-horizons/unknown-horizons/blob/master/horizons/constants.py#L41-L89
- so it return "unknown" or default
- then version comparison "from server" with "unknown local version" results in wrong result
  - ref. https://github.com/unknown-horizons/unknown-horizons/blob/master/horizons/util/checkupdates.py#L62-L73
- For Linux setup.py creates: gitversion.txt
  - ref. https://github.com/unknown-horizons/unknown-horizons/blob/master/setup.py#L51-L54
- The Windows installer doesn't install such a file.

= Install gitversion.txt with the windows installer
= When the gitversion.txt is not found, fallback to the hardcoded version.
